helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -97,7 +102,9 @@ data:
         Systemd_Filter    SYSLOG_IDENTIFIER=sshd
         Systemd_Filter    _SYSTEMD_UNIT=sshd.service
         threaded          on
+    {{- end }}
```

Comment:
This end is related to line 92

Reply:
oh, my bad, it's because of the `sshd` / `execve` split! I see it now! :+1:

Reply:
:)